scripts/run_endpoint.sh

```diff
@@ -81,5 +99,10 @@ elif [ "$ROLE" == "server" ]; then
     esac
 
     quicinteropserver ${SERVER_PARAMS} -root:/www -listen:* -port:443 \
-        -file:/certs/cert.pem -key:/certs/priv.key 2>&1
+        -file:/certs/cert.pem -key:/certs/priv.key &
```

Comment:
For some reason, the `&` doesn't seem to be working as expected. The `getchar()` call in there isn't blocking as expected. Anyone have any ideas why that'd be?

Reply:
I know we ran into the same issue in the perf setup, where when running in the background stdin wouldn't block. We solved that with the udp explicit shutdown function.

Reply:
Ok. Just pushed a possible solution (`-noexit`).